markets/storageadapter/dealpublisher_test.go

```diff
@@ -186,7 +187,7 @@ func TestDealPublisher(t *testing.T) {
 	}
 }
 
-func TestForcePublish(t *testing.T) {
+func TestPublishPendingDeals(t *testing.T) {
```

Comment:
Let's also add a test for publishing less than all deals.
eg publish only 1 of 2 pending deals.